app/client/src/redux/ducks/communityDuck.js

```diff
@@ -75,7 +77,7 @@ export const createCommunity =
         body: JSON.stringify({
           title: communityData.title,
           description: communityData.description,
-          creator: communityData.creator,
+          creator: name,
```

Comment:
When you create a community or a post you don't specify the creator in the request body. It grabs that from the authentication